resources/js/stores/ModalsState.ts

```diff
@@ -75,5 +76,17 @@ export const useTogglablesStateStore = defineStore("togglables-store", {
 				// });
 			}
 		},
+
+		rememberScrollThumb(photo_id: string | undefined) {
+			this.scroll_photo_id = photo_id;
+		},
+
+		recoverAndResetScrollThumb(thumbElem: HTMLElement) {
+			if (thumbElem) {
+				thumbElem.scrollIntoView();
+			}
+
+			this.scroll_photo_id = undefined;
```

Comment:
I see, whenever you get out of the photo view, you clear out position.
That way you do not need to keep the state per album visited. Smart.